### ISSUE TITLE
Welt-Generierung 

### ISSUE THREAD

**Issue by DerOliDE** [combak/ShatteredWorld2#4]
Das übergeordnete Thema "Ozean" steht ja, passend zum 1.13er Update, schon so grob fest. Soll es dabei eine zufällig generierte Karte geben oder findet sich wieder jemand der eine liebevoll mit dem Open Terrain Generator erstellt?

Soll es nur Ozean geben (Wasser bis 255 Höhe) oder sollen auch Objekte auf der Wasseroberfläche baubar sein?

**Comment by Grambosch** [comment]
Geht es den ein beliebige Wasserhöhe einzustellen? Wenn ja würd ich die ja irgendwo bei 160-200 setzen und der Rest der Welt darunter kann komplett random generiert sein

**Comment by DerOliDE** [comment]
Ich weiß jetzt nicht aus dem Stegreif, ob es Open Terrain Gen kann, aber mit einer Mod bzw. eigenen Worldgen wäre das machbar.

**Comment by Tr4pp3r** [comment]
Ja, das Wasserlevel kann man einstellen. Wir müssen uns dann nur anschauen, wie das mit den anderen Biomen zusammenpasst. Bei AoT hatte ich da immer wieder Schwierigkeiten, da lief das ganze aber auch noch über TerrainControl. Ich weiß, dass die Entwickler das noch auf ihrer ToDo-Liste hatten bin gerade aber nicht up-to-date ob da dran gearbeitet wurde. Evtl ist's mit OTG jetzt angenehmer. Es gibt aber auch noch andere Möglichkeiten Karten zu gestalten. Die Frage ist, wie weit in die Generierung der Erze etc  eingegriffen werden soll. Da gibt's dann noch einiges zu klären, aber wir schaffen das schon.
Das Karte(n) zeichnen würde ich dann auch gerne wieder übernehmen.

**Comment by hellamie** [comment]
Unter Wasser Start? oder auf einer kleinen kleinen Insel?

bin für Unterwasser Dome, wie in dem Raumschiff Pack.

**Comment by CD4017BE** [comment]
Vielleicht Meeresspiegel bei 192, Ozeangrund größtenteils variierend im Bereich 64-96, Plateaus bei 120-160 und sehr seltene Inseln/Kontinente, die bis an die Oberfläche reichen (Terrain-Erhebungen um die 200, Hügel/Berge bis 240 maybe).

**Comment by DerOliDE** [comment]
So mal ein weiterer Gedankenfurz alles komplett Unterwasser mit 3 Tiefenregionen
1) 256 - 128 - Kaum Druck - Benötigt nur Ausrüstung für Sauerstoff.
2) 127-64 Mittler Druck - Benötigt zusätzlich Ausrüstung gegen den dort herrschenden Druck.
3) 63-0 Starker Druck - Nur mit Hight Tech Ausrüstung überlebbar.
